src/pages/TextFieldDemo/TextFileDemo.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import TextField from '../../components/index';
+import { Paragraph } from '../../components/TextField/style';
+
+const TextEditor = () => (
+  <>
+    <Paragraph>This is a disable input</Paragraph>
+    <TextField value="Disabled Input" disabled="true" />
```

Comment:
for pass boolean true value we didn't use the value in string just pass disabled as props like (disabled)

Reply:
Done